docs/en/reference/improving-performance.rst

```diff
@@ -61,6 +65,7 @@ Apply Best Practices
 A lot of the points mentioned in the Best Practices chapter will
 also positively affect the performance of Doctrine.
 
+See :ref:`best-practices`
```

Comment:
I'm not very familiar with rst, so I read the docs, shouldn't it be `:doc:` instead of `:ref`?

Reply:
Mhh, I followed the `:ref` documentation (http://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#cross-referencing-objects) which says:

> Using ref is advised over standard reStructuredText links to sections (like `Section title`_) because it works across files, when section headings are changed, will raise warnings if incorrect, and works for all builders that support cross-references. 

But you are right, I link to a document and not to a specific section inside it. So I will change it.

Reply:
Can you test your changes locally? I'm not an expert in rst, so it's best if you just generate the docs and open them with a browser.

Reply:
I created the documentation locally and it worked the new links worked for me.